packages/react-components/style/trace-explorer.css

```diff
@@ -315,17 +315,34 @@
     font-size: var(--trace-viewer-ui-font-size1);
     min-height: 23px;
     white-space: nowrap;
+    font-weight: bold;
 }
 
-.shortcuts-table i {
-    color: var(--trace-viewer-button-background);
+.shortcuts-icon {
+    color: var(--theia-button-background);
```

Comment:
Please rebase the code. CSS variables that start with `--theia` should be replaced by `--trace-viewer`.